README.md

```diff
@@ -33,3 +33,9 @@ Here's an example for [Ruby's Isogram](https://github.com/exercism/website-copy/
 ![Create a new file](http://g.recordit.co/96zCDgEkfG.gif)
 
 </details>
+
+### Facts 
+
+Software developer Katrina Owen created Exercism while she was teaching programming at Jumpstart Labs.The platform was developed as an internal tool to solve the problem of her own students not receiving feedback on the coding problems they were practicing. Katrina put the site publicly online and found that people were sharing it with their friends, practicing together and giving each other feedback. Within 12 months, the site had organically grown to see over 6,000 users had submitted code or feedback, and hundreds of volunteers contribute to the languages or tooling on the platform.
```

Comment:
```suggestion
Software developer Katrina Owen created Exercism while she was teaching programming at Jumpstart Labs. The platform was developed as an internal tool to solve the problem of her own students not receiving feedback on the coding problems they were practicing. Katrina put the site publicly online and found that people were sharing it with their friends, practicing together and giving each other feedback. Within 12 months, the site had organically grown to see over 6,000 users had submitted code or feedback, and hundreds of volunteers contribute to the languages or tooling on the platform.
```

Interesting.

There was a missing space after the end of a sentence.

I am not sure if this is the place for the introductory or if "Facts" is a good heading, or if that should be a subheading under Mentor notes.  But I like the history point.